### ISSUE TITLE
Message Center: Button text not visible

### ISSUE THREAD

**Issue by faizfolio** [apptentive/apptentive-android#188]
Hi,
We are using Message Center feature in our app. We came to notice that the button text color is not visible properly and also not customizable from theme override. 

We are using material design theme in our app.

Apptentive library version: 5.4.5

![Android_Feedback](https://user-images.githubusercontent.com/42534213/65034438-7a182000-d960-11e9-92fc-d17e1dad5e18.JPEG)

**Comment by weeeBox** [comment]
Hey @faizfolio,

We're working on migrating our UI to material design components. In a mean time you can fix button style with this snippet (inside your theme file):

```
    <style name="ApptentiveButtonStyle" parent="Apptentive.Widget.Button.Borderless">
        <item name="android:textColor">@color/colorPrimary</item>
    </style>

    <style name="ApptentiveThemeOverride">
        ...
        <item name="apptentiveInteractionButtonStyle">@style/ApptentiveButtonStyle</item>
    </style>
```

**Comment by harisf3** [comment]
This still remains same. when we can expect new library?
![PPA-2208_3 5 1_Failed](https://user-images.githubusercontent.com/43817946/66109971-30296e00-e5df-11e9-9218-257d5cbbf0f6.JPEG)

**Comment by CaseyApptentive** [comment]
Hi @harisf3 ! Let me sync with my team to get you an answer. 

Stay tuned,

Casey

**Comment by CaseyApptentive** [comment]
Hi @harisf3,

Thanks again for this new report. Did the override that @weeeBox provided previously fix the other button style issue for you? Let me know as we're working to prioritize material design changes more broadly. 

Also, could you email me directly at support@apptentive.com and let me know which specific app you're working on? I'd like to keep your Customer Success Manager in the loop as we work on a solution.

Thanks!

Casey

**Comment by harisf3** [comment]
Yup the previous override kinda fixed the issue, it showed black test over blue background, which is not ideal but better than invisible at all.

**Comment by samtipton** [comment]
Having the same problem as what @harisf3 reported on October 3rd. Any updates here? I'm on v5.5.0, androidx 1.0.0, targeting 28 building on Android 10

**Comment by CaseyApptentive** [comment]
Hi @samtipton, sorry that you're seeing the same issue. I'll work with my Android engineers to get you a more complete answer. 

Apologies if this takes longer than usual over the holidays. 

Feel free to reach us at support@apptentive.com if you have any further questions, otherwise I'll follow up here when I know more.

**Comment by drivardST** [comment]
Any update on this issue @CaseyApptentive? I'm following up on old tickets in our project.

**Comment by kikin81** [comment]
@CaseyApptentive please take a look at a related issue with hardcoded values for the buttons in the dialog here:

https://github.com/apptentive/apptentive-android/issues/202

In general it's bad practice to hardcode button heights, specially when they are not compliant with Google Accessibility minimum requirements of *48 dp* https://support.google.com/accessibility/android/answer/7101858?hl=en

**Comment by CaseyApptentive** [comment]
Hi everyone. Apologies for the delay. We're working on migrating to material design components for future releases, including some other large improvements, for a forthcoming SDK. I've shared this issue with my product team. 

Is this blocking a release? Let me know as it will help prioritize this on our roadmap.
